### PR TITLE
Cleaning up rehashing logic as it is a dead code as of now.

### PR DESCRIPTION
**Pull request by raghumdani**
[Motivation] #165

### CODE REVIEW

File: deltacat/__init__.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
 deltacat.logs.configure_deltacat_logger(logging.getLogger(__name__))
 
-__version__ = "0.1.18b12"
```
Comment (pfaraone) [inline deltacat/__init__.py:46]: Version number HEAD is 0.1.18b12

Comment (pfaraone) [reply deltacat/__init__.py:46]: LGTM apart from the version number

Comment (raghumdani) [reply deltacat/__init__.py]: Fixed.

**Review by pfaraone — APPROVED** [review]
LGTM

**Pull request merged**